ibm/service/cdtektonpipeline/resource_ibm_cd_tekton_pipeline.go

```diff
@@ -401,6 +401,12 @@ func ResourceIBMCdTektonPipeline() *schema.Resource {
 							Computed:    true,
 							Description: "Mark the trigger as a favorite.",
 						},
+						"limit_waiting_runs": &schema.Schema{
```

Comment:
triggers is computed, why it has optional fields inside it ?

Reply:
These fields may or may not be present in the trigger output, but if they are present they are computed. This is what the SDK generator generates for us

Reply:
Updated to remove optional for limit_waiting_runs